### PR TITLE
Add common vector constants to JS

### PR DESCRIPTION
**Pull request by jherico**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: fbdf7f3b14daead18fba6d24795beebccf9d1294 available at: https://gist.github.com/568d4ff3435ea5f4d502<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5677.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5677.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 868d57e886857931b7885159026cbeaa19b042de available at: https://gist.github.com/568d4ff3435ea5f4d502<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5677.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5677.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: ea672d80ccc944624e37a101280ab9fcffa08584 available at: https://gist.github.com/568d4ff3435ea5f4d502<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5677.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5677.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 2aa1976b5d470e38129bdb1d329dd1a5d0b7b9b5 available at: https://gist.github.com/568d4ff3435ea5f4d502<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5677.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5677.dmg

File: libraries/script-engine/src/Vec3.cpp [hunk 1]
```diff
@@ -123,3 +123,70 @@ glm::vec3 Vec3::fromPolar(float elevation, float azimuth) {
     return fromPolar(v);
 }
 
+const vec3& Vec3::UNIT_X() {
```
Comment (ZappoMan) [inline libraries/script-engine/src/Vec3.cpp]: won't block merge, but why not make these all inline in the header... seems like that would be appropriate

**Pull request merged**